README.md

```
@@ -1,2 +1,75 @@
+## Features
+
+#### Lights: 
+- [x] Directional lights
+- [x] Ambient lights
+- [x] Point lights
+- [x] Diffuse lights
+- [x] Colored lights
+- [x] Multiple point lights
+- [x] Drop shadows
+
+#### Primitives:
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)